data/sles4sap/qe_sap_deployment/sles4sap_azure_generic_maintenance.yaml

```diff
@@ -0,0 +1,41 @@
+# SUSE's openQA tests
+#
+# Copyright SUSE LLC
+# SPDX-License-Identifier: FSFAP
+# Maintainer: QE-SAP <qe-sap@suse.de>
+# Summary: Generic yaml template for use with qe-sap-deployment project: https://github.com/SUSE/qe-sap-deployment
+#   Settings are meant to be controlled via OpenQA variables and managed by test:
+#   tests/sles4sap/publiccloud/qesap_ansible.pm
+
+provider: 'azure'
+apiver: 3
+terraform:
+  variables:
+    # GENERAL VARIABLES #
+    vnet_address_range: "%VNET_ADDRESS_RANGE%"
```

Comment:
This new file seems https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/data/sles4sap/qe_sap_deployment/sles4sap_azure_generic.yaml with 2 differences:

- Two additional terraform variables: `vnet_address_range` and `subnet_address_range`. But maybe we can include them directly in the sles4sap_azure_generic.yaml and just add a default or an additional couple of settings in the job group using  sles4sap_azure_generic.yaml

- `os_image` is using `"%PUBLIC_CLOUD_OS_IMAGE%"`  where the sles4sap_azure_generic.yaml is using `"%PUBLIC_CLOUD_IMAGE_ID%"` . Why we have this difference?

All in all: is it possible not to duplicate this file and directly use sles4sap_azure_generic.yaml

Reply:
I'm ok to do it in a next PR: "Optimize the test to reuse sles4sap_azure_generic.yaml"